bin/yk-config

```diff
@@ -77,32 +34,21 @@ handle_arg() {
     fi
 
     case $1 in
-        --ao) set_aot_pipeline $2;;
         --cc) OUTPUT="${ykllvm_bin_dir}/clang" ;;
         --cxx) OUTPUT="${ykllvm_bin_dir}/clang++" ;;
         --ar) OUTPUT="${ykllvm_bin_dir}/llvm-ar" ;;
         --ranlib) OUTPUT="${ykllvm_bin_dir}/llvm-ranlib" ;;
         --cflags)
-            # FIXME: we can only do -O0 for now.
-            OUTPUT="${OUTPUT} -O0"
             # Enable LTO.
             OUTPUT="${OUTPUT} -flto"
             # Outline functions containing loops during AOT compilation. Needed
             # for `yk_unroll_safe`.
             OUTPUT="${OUTPUT} -fyk-noinline-funcs-with-loops"
-            # Required so that our curated set of optimisation passes can run.
-            # Without this, clang will slap `optnone` attributes on every
-            # function, causing optimisations to skip them.
-            OUTPUT="${OUTPUT} -Xclang -disable-O0-optnone"
-            # Run these "prelink" optimisation passes.
-            #
-            # These are run on the individual pre-LTO-merged LLVM modules.
-            if [ ! -z "${PRELINK_PASSES}" ]; then
-                OUTPUT="${OUTPUT} -mllvm --newpm-passes=${PRELINK_PASSES}"
-            fi
-            case $profile in
-                debug*) OUTPUT="$OUTPUT -g" ;;
-            esac
+
+            # Disable backend optimisations that Yk can't handle by adding the
```

Comment:
Just noticed a mistake here. We have a linker flag in the --cflags section!

Will fix and retest.

Reply:
Fixed in [d874f9c](https://github.com/ykjit/yk/pull/1411/commits/d874f9cd2a4d0c9b67317ff4accf5e8991ee81a7)